.github/workflows/test.yml

```diff
@@ -72,9 +72,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - name: CMake Configure
-        run: cmake -B out -DCMAKE_BUILD_TYPE=Release ${{ runner.os == 'macOS' && '-DBASE64_WITH_AVX2=OFF' || '' }}
+        run: cmake -B out -Werror=dev -DBASE64_BUILD_TESTS=ON -DCMAKE_BUILD_TYPE=Release
```

Comment:
Why did the conditional statement disappear?

Reply:
Ah, it was moved to line 41. Nevermind.

Reply:
alpine is always linux, no need for any conditional here.